scvi/module/_amortizedlda.py

```diff
@@ -200,7 +200,9 @@ def forward(
         with pyro.plate(
             "cells", size=n_obs or self.n_obs, subsample_size=x.shape[0]
         ), poutine.scale(None, kl_weight):
-            cell_topic_posterior_mu, cell_topic_posterior_sigma, _ = self.encoder(x)
+            cell_topic_posterior, _ = self.encoder(x)
+            cell_topic_posterior_mu = cell_topic_posterior.loc
+            cell_topic_posterior_sigma = 2.0 * cell_topic_posterior.scale.log()
```

Comment:
What's the reason for this? For equivalence to what it was before wouldn't it just be `cell_topic_posterior.scale**2`?

Reply:
You're right. I thought that `cell_topic_posterior_sigma` was the log-variance here. 
However, on an unrelated note to this PR, this means that in this model the posterior variance is lower bounded by `Softplus(0.0) ~ 0.6931` though. Why is that?

Reply:
Yea that's a good point. I think was using  FC Layers at one point then switched it out so I had assumed the output var value could be negative. Even with it being constrained to be positive, I think I had to do this because Pyro would try stepping into a negative value and plugging it into the Normal distribution, so softplussing it prevented that. I can make an issue for this